Source/WTF/wtf/URL.h

```diff
@@ -259,7 +259,7 @@ WTF_EXPORT_PRIVATE bool protocolHostAndPortAreEqual(const URL&, const URL&);
 WTF_EXPORT_PRIVATE Vector<KeyValuePair<String, String>> differingQueryParameters(const URL&, const URL&);
 WTF_EXPORT_PRIVATE Vector<KeyValuePair<String, String>> queryParameters(const URL&);
 WTF_EXPORT_PRIVATE bool isEqualIgnoringQueryAndFragments(const URL&, const URL&);
-WTF_EXPORT_PRIVATE void removeQueryParameters(URL&, const HashSet<String>&);
+WTF_EXPORT_PRIVATE Vector<String> removeQueryParameters(URL&, const HashSet<String>&);
```

Comment:
Maybe a comment to explain what this returns exactly.

The fact that it may return duplicates for e.g. should probably be documented too.

Reply:
Good point — I'll add a comment explaining this:

```
// Returns the parameters that were removed (including duplicates), in the order that they appear in the URL.
```